eg/esc-bidirectional.js

```diff
@@ -0,0 +1,30 @@
+var five = require("../");
+var board = new five.Board();
+
+board.on("ready", function() {
+  var start = Date.now();
+  var esc = new five.ESC({
+    type: "bidirectional",
```

Comment:
Alternatively...


```js
// Assume...
// ESC.F = 0, Forward (default)
// ESC.FR = 1, Forward/Reverse
// ESC.FBR = 2, Forward/Brake/Reverse (doubt this can be supported, but included for illustration)


var esc = new five.ESC({
  mode: ESC.FR, 
  neutral: 50,
  pin: 11
});
```